Lib/dis.py

```diff
@@ -32,7 +32,8 @@ def _try_compile(source, name):
     return c
 
 def dis(x=None, *, file=None, depth=None):
-    """Disassemble classes, methods, functions, generators, or code.
+    """Disassemble classes, methods, functions, generators,
```

Comment:
Perhaps:

    """Disassemble classes, functions, and other compiled objects (e.g. generators)"""